packages/vite-plugin-cloudflare/playground/react-spa/__tests__/assets.spec.ts

```diff
@@ -20,3 +22,22 @@ test("returns the home page for not found routes", async () => {
 	const content = await page.textContent("h1");
 	expect(content).toBe("Vite + React");
 });
+
+test("applies _headers", async ({}) => {
+	const response = await fetch(viteTestUrl);
+	expect(response.headers.get("X-Header")).toBe("Custom-Value!!!");
+});
+
+test("applies _redirects", async ({}) => {
+	const response = await fetch(`${viteTestUrl}/foo`, { redirect: "manual" });
+	expect(response.status).toBe(302);
+	expect(response.headers.get("Location")).toBe("/bar");
+});
```

Comment:
Redirects are only being applied if the path doesn't match an existing non-HTML asset so this also wouldn't pass for other asset types.

Reply:
And this one too